client/packages/system/src/Encounter/DetailView/SidePanel.tsx

```diff
@@ -35,6 +36,9 @@ export const SidePanel: FC<SidePanelProps> = ({ encounter, onChange }) => {
   const [encounterNote, setEncounterNote] = useState(
     encounter.document.data?.notes?.[0]?.text ?? ''
   );
+  const [createdBy, setCreatedBy] = useState(
+    encounter?.document?.data?.createdBy.username ?? ''
```

Comment:
```suggestion
    encounter?.document?.data?.createdBy?.username ?? ''
```

Reply:
I don't think that's necessary for two reasons:
1. the `createdBy` object should always have `id` and `username` in it (based on how it's created)
2. but even if it didn't, you don't need a `?.` for the final property in a path, as it'll just return undefined if not present (or in this case the fallback `''`). The `?.` operator is just to prevent you having to do `data && data.createdBy.username` for every higher property in the path.

Reply:
🤔 I thought so too... but it wouldn't load encounters from the init data for me without that extra optional chaining (?)

Reply:
Oh really? Okay, I'll add it in just to be sure.

Reply:
efec5e4fc